scripts/bash-comments-lint.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import logging, sys, re, os
+
+LINE_LENGTH = 80
+
+def lint(line):
+    new_line = ""
+    end_of_line = ""
+
+    # Getting rid of the new line
+    if line[-1] == "\n":
+        end_of_line = "\n"
+
+    core = " ".join(re.findall("\w+", line))
+    start = "# "
+    place = LINE_LENGTH - (len(start) + 1 + len(core) + 1)
+    before = "-"*(place//2) + " "
+    after = " " + "-"*((place+1)//2)
+    new_line = start + before + core + after + end_of_line
+
+    line = line.replace("\n","(NEW LINE)")
+    new_line = new_line.replace("\n","(NEW LINE)")
+    print("linter suggestion: \n"
+          "```\n"
+          f"-{line}\n"
+          f"+{new_line}\n"
+          "```")
+    ask_user = input("Approve change? (y/n)")
+    return False if ask_user=="n" else new_line.replace("(NEW LINE)", "\n")
+
+
+def main(script):
+    write = False
+    datas = []
+    with open(script, "r", encoding="utf-8") as file:
+        datas = file.readlines()
+        for n, line in enumerate(datas):
+            if re.search("^# -",line):
+                new_line = lint(line)
+                if new_line:
+                    write = True
+                    datas[n] = new_line
+    if write:
+        with open(script, "w", encoding="utf-8") as file:
+            file.writelines(datas)
+
+
+if __name__ == "__main__":
```

Comment:
and here you could add a little `--help` with `argparse` as well, explaining the `--line-length` option and the purpose of the script
also an option to avoid asking confirmation for each line, that kind of stuff :smirk:

Reply:
see [32c9a2f]()

Reply:
:+1: